ichnaea/rate_limit.py

```diff
@@ -0,0 +1,12 @@
+from redis import ConnectionError
+
+
+def rate_limit(redis_client, key, maxreq=0, expire=86400):
+    if maxreq:
+        try:
+            redis_client.set(key, 0, ex=expire, nx=True)
+            count = redis_client.incr(key, 1)
+            return count > maxreq
```

Comment:
Oh that's so weird, I tried doing that and it was raising an error, but I just tried again and it's magically not, maybe I was doing something differently.  Yes your'e right.